_posts/shwetashambhavi/2020-10-21-dictionary-pair-programming-shwetashambhavi.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+author1: shwetashambbhavi
+author2: EAkita
```

Comment:
This still isn't the format our website will be looking for. Take a look at the assignment to compare.